PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBUser.h

```diff
@@ -58,6 +58,9 @@ NS_ASSUME_NONNULL_BEGIN
 // Placeholder for exchange-specific extensions to OpenRTB.
 @property (nonatomic, strong, nullable) NSMutableDictionary<NSString *, NSObject *> *ext;
 
+//ID of the user within the app.
```

Comment:
Let's use the OpenRTB description for this property - `Exchange-specific ID for the user.`